pipelines/controller/config/application.yaml

```diff
@@ -69,7 +71,7 @@ fhirdata:
   # Equivalent to pipeline `resourceList` parameter.
   # Note there is no Questionnaire in our test FHIR server, but it is okay; see
   # https://github.com/google/fhir-data-pipes/issues/785.
-  resourceList: "Patient,Encounter,Observation,Questionnaire"
+  resourceList: "Patient,Encounter,Observation,Questionnaire,Condition,Practitioner,Location,Organization"
```

Comment:
To be consistent, can we have this change at `docker/config/application.yaml` as well?

Reply:
Done. I originally changed this only here (i.e., for dev. env.) because the default `Patient,Encounter,Observation` is referenced in several places (e.g., Wiki, `FhirEtlOptions` etc). But I think that is fine to keep these consistent (I'll update the Wiki later).